.github/workflows/e2ePerformanceTests.yml

```
@@ -0,0 +1,158 @@
+name: E2E Performance Tests
+
+on:
+  workflow_call:
+    inputs:
+      DELTA_REF:
+        description: A branch or commit ref to build the test build on
+        type: string
+        required: true
+      PR_NUMBER:
+        description: The number of a merged PR, if this workflow is being triggered by a PR merge
+        type: string
+        required: true
+
+  workflow_dispatch:
+    inputs:
+      DELTA_REF:
+        description: A branch or commit ref to build the test build on
+        type: string
+        required: true
+
+jobs:
+  getMostRecentRelease:
+    runs-on: ubuntu-latest
+    name: Get the most recent release version
+    outputs:
+      VERSION: ${{ steps.getMostRecentRelease.outputs.VERSION }}
+    steps:
+      - name: Get most recent release version
+        id: getMostRecentRelease
+        run: echo "VERSION=$(gh release list --limit 1 | awk '{ print $1 }')" >> "$GITHUB_OUTPUT"
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+
+  buildBaseline:
+    runs-on: ubuntu-20.04-64core
+    needs: [getMostRecentRelease]
+    name: Build apk from latest release as a baseline
+    steps:
+      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+        with:
+          fetch-depth: 0
+
+      - name: Check if there's an existing artifact for this baseline
+        id: checkForExistingArtifact
+        uses: xSAVIKx/artifact-exists-action@3c5206b1411c0d2fc0840f56b7140646933d9d6a
+        with:
+          name: baseline-apk-${{ needs.getMostRecentRelease.outputs.VERSION }}
+
+      - if: fromJSON(steps.checkForExistingArtifact.outputs.exists)
+        run: echo 'APK for baseline ${{ needs.getMostRecentRelease.outputs.VERSION }} already exists, reusing existing build'
+
+      - name: Checkout "Baseline" commit (last release)
+        if: !fromJSON(steps.checkForExistingArtifact.outputs.exists)
```

I think this needs to be an expression

```suggestion
        if: ${{ !fromJSON(steps.checkForExistingArtifact.outputs.exists) }}
```

---

Technically not required in an `if` statement but yeah we should for consistency